panel/pane/plot.py

```diff
@@ -210,7 +221,7 @@ class Matplotlib(Image, IPyWidget):
     dpi = param.Integer(default=144, bounds=(1, None), doc="""
         Scales the dpi of the matplotlib figure.""")
 
-    encode = param.Boolean(default=True, doc="""
+    encode = param.Boolean(default=False, doc="""
```

Comment:
I believe (?) encode was set to `True` to try to enable responsive 'svg' outputs. I believe the issues have been fixed and encoding is no longer needed. And encoding slows things down so I believe it should not be the default.

Please also note the `encode` parameter was not documented in the reference Notebook and I've never met anyone on discourse asking about it. So I believe its safe to change.

Reply:
Agreed.